governance/polls/poll-parameters/goerli-approval-budget-example.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Approval Voting with Majority Budget Example
+summary: Signal your support or opposition for the DUX CU Budget Options.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp52-development-ux-core-unit-budget-dux-001/12085
+parameters:
+  input_format: 
+    type: choose-free
+    options: [1,2,3]
```

Comment:
if you want option 3 not to be able to be selected at the same time than option 1, 2, you should put "options": [1,2]

Reply:
In this scenario, there may be a voter who would approve the current budget, or reject, but not an increased budget which is why I left it in as an option